### ISSUE TITLE
Errors UFO's

### ISSUE THREAD

**Issue by danietech** [shenxn/protonmail-bridge-docker#58]
Hi Project leaders,

Small introduction, I am a hobbyist, and I would find this docker very useful, however, I feel my experience may be lacking to complete the setup successfully just to list out:

I have loaded the docker with ports as mentioned -p 1025 -p 1143
According to the setup, I must enter the command on the first execution

" docker exec -it protonmail-bridge /bin/bash
chmod +x entrypoint.sh
./entrypoint.sh init "

I am not too sure what this is declaring via the console, however, I have managed to enter this in on the Unraid terminal and it initialises the protonmail prompt

I found that I am unable to login directly after entering the login command but I found a workaround on this website below:

https://www.hagensieker.com/wordpress/2021/11/18/protonmail-bridge-on-unraid/

helpful, it states I call to the "top" and kill the service and then call to command "protonmail-bridge --cli"

and this restarts the proton mail bridge as a client I think.

From here I use the login command and proceed to login to protonmail entering my user name and password and the twoFA

and for the most part, this seems straight forward but on observation, after this stage, I get a series of UFOs, please see below

--------------------------------------
ProtonMail Bridge is not able to detect a supported password manager
(secret-service or pass). Please install and set up a supported password manager
and restart the application.
FATA[May 16 19:30:34.257] Failed to launch                              error="signal: terminated"
+ [[ '' == init ]]
+ socat TCP-LISTEN:25,fork TCP:127.0.0.1:1025
+ rm -f faketty
+ socat TCP-LISTEN:143,fork TCP:127.0.0.1:1143
+ mkfifo faketty
+ cat faketty
+ protonmail-bridge --cli
time="2022-05-16T19:41:24Z" level=warning msg="Failed to add test credentials to keychain" error="failed to open dbus connection: exec: \"dbus-launch\": executable file not found in $PATH" helper="*keychain.SecretServiceDBusHelper"
time="2022-05-16T19:41:24Z" level=warning msg="Failed to add test credentials to keychain" error="pass not initialized: exit status 1: Error: password store is empty. Try \"pass init\".\n" helper="*pass.Pass"
ERRO[May 16 19:41:24.346] Could not list credentials                    error="no keychain" pkg=credentials
ERRO[May 16 19:41:24.347] Could not load all users from credentials store  error="no keychain" pkg=users
--------------------------------------

To note I am unable to identify what the above means but I am aware this is not how the program should run.

To summarise I have added an account successfully before as I needed it to use for another Docker on the same network.
I am trying to use this to my nextcloud docker and this is a strict HTTPS, I was wondering if it is being denied but I think this may be due to the above errors on setup.

Thanks if you have read this far, if there is any help that can stir me in the right direction, I would be very grateful 

D.

**Comment by shenxn** [comment]
Sorry for the late reply. To me it seems that the password store `pass` is not initialized. But it should be when you run `init`. Can you check if your volume mount is configured correctly?

**Comment by bozzio83** [comment]
Same issue here.  What do you mean by volume mount?

**Comment by WNiels** [comment]
Any updates on this? I think i have the same/similar issue running this on unraid. I also followed this guide: https://hagensieker.com/2021/11/18/protonmail-bridge-on-unraid/


    root@xxx:/protonmail# protonmail-bridge --cli
    INFO[0000] Migrating keychain helper                    
    WARN[Jan 19 15:42:11.092] Failed to add test credentials to keychain    error="failed to open dbus connection: exec: \"dbus-launch\": executable file not found in $PATH" helper="*keychain.SecretServiceDBusHelper"
    WARN[Jan 19 15:42:11.092] Keychain is not available.                    keychain=SecretServiceDBus
    WARN[Jan 19 15:42:11.092] Keychain is not available.                    keychain=SecretService
    
                Welcome to Proton Mail Bridge interactive shell
                                  ___....___
        ^^                __..-:'':__:..:__:'':-..__
                      _.-:__:.-:'':  :  :  :'':-.:__:-._
                    .':.-:  :  :  :  :  :  :  :  :  :._:'.
                 _ :.':  :  :  :  :  :  :  :  :  :  :  :'.: _
                [ ]:  :  :  :  :  :  :  :  :  :  :  :  :  :[ ]
                [ ]:  :  :  :  :  :  :  :  :  :  :  :  :  :[ ]
       :::::::::[ ]:__:__:__:__:__:__:__:__:__:__:__:__:__:[ ]:::::::::::
       !!!!!!!!![ ]!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!![ ]!!!!!!!!!!!
       ^^^^^^^^^[ ]^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^[ ]^^^^^^^^^^^
                [ ]                                        [ ]
                [ ]                                        [ ]
          jgs   [ ]                                        [ ]
        ~~^_~^~/   \~^-~^~ _~^-~_^~-^~_^~~-^~_~^~-~_~-^~_^/   \~^ ~~_ ^
    >>> ^C
    FATA[Jan 19 15:42:53.315] Failed to launch                              error="exit status 1" exe_path=/usr/lib/protonmail/bridge/bridge exe_to_launch=bridge launcher_path=/usr/lib/protonmail/bridge/proton-bridge launcher_version=3.8.1
    root@xxx:/protonmail# pass
    Password Store
    `-- protonmail-credentials
        `-- xxxxxxxxxxx==
            `-- bridge-vault-key
    root@xxx:/protonmail#

**Comment by thorin29** [comment]
Same issue on Unraid today, was working fine with all my other containers, had to restart my containers and now I receive this message.  The container runs but is causing issues with other containers that rely on smtp.


![Error Message](https://github.com/shenxn/protonmail-bridge-docker/assets/97200048/c15bb65c-eeb2-46e7-8337-a7187c71f94c)

**Comment by simonfelding** [comment]
`INFO[0000] Migrating keychain helper    ` indicates it's related to protonmail v2 -> v3. You need to re-initialize because they messed some stuff up basically.
